LLama/LLamaEmbedder.cs

```diff
@@ -65,10 +65,6 @@ public float[] GetEmbeddings(string text)
         /// <exception cref="RuntimeError"></exception>
         public float[] GetEmbeddings(string text, bool addBos)
         {
-            if (addBos)
-            {
-                text = text.Insert(0, " ");
```

Comment:
do we really want to remove this? I had a load of weird issues one time when I didn't add a leading space to inputs.